src/DataStructures/Tensor/TypeAliases.hpp

```diff
@@ -334,11 +334,15 @@ using ijaa = Tensor<DataType, tmpl::integral_list<std::int32_t, 3, 2, 1, 1>,
                                SpatialIndex<SpatialDim, UpLo::Lo, Fr>,
                                SpacetimeIndex<SpatialDim, UpLo::Lo, Fr>,
                                SpacetimeIndex<SpatialDim, UpLo::Lo, Fr>>>;
-
 }  // namespace tnsr
 
 template <size_t Dim, typename Frame1, typename Frame2>
 using InverseJacobian =
     Tensor<DataVector, tmpl::integral_list<std::int32_t, 2, 1>,
            typelist<SpatialIndex<Dim, UpLo::Up, Frame1>,
                     SpatialIndex<Dim, UpLo::Lo, Frame2>>>;
+
+template <size_t Dim, typename Frame1, typename Frame2>
```

Comment:
I think we should use SourceFrame and TargetFrame as the template parameters (and above in InverseJacobian) to match CoordinateMap

Reply:
I agree, `Frame1` -> `SourceFrame` and `Frame2` -> `TargetFrame`?